commit 27227d289339c63277091f84592fa0bd388b0ea5 (refs/changes/55/2955/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-16T10:40:04-08:00 (1 year, 11 months ago)
    
    ZAPI-811 vms.volumes.tests stalls in nightly-1 and makes tests suite time out

diff --git a/test/vms.volumes.test.js b/test/vms.volumes.test.js
index e2f3cc9..7351f6e 100644
--- a/test/vms.volumes.test.js
+++ b/test/vms.volumes.test.js
@@ -46,7 +46,11 @@ function getVmPayloadTemplate() {
         image_uuid: VMAPI_ORIGIN_IMAGE_UUID,
         server_uuid: SERVER.uuid,
         networks: [ { uuid: ADMIN_FABRIC_NETWORK.uuid } ],
-        brand: 'joyent-minimal',
+        /*
+         * We use the 'joyent' brand on purpose here since joyent-minimal
+         * doesn't support mounting NFS volumes.
+         */
+        brand: 'joyent',
         billing_id: '00000000-0000-0000-0000-000000000000',
         ram: 64,
         quota: 10,
@@ -248,7 +252,10 @@ exports.create_vm_with_valid_volumes_params = testIfVolapiPresent(function (t) {
 
                     waitForValue('/jobs/' + vmProvisioningJobUuid, 'execution',
                         'succeeded',
-                        { client: client, timeout: 4 * 60 * 1000 },
+                        /*
+                         * 4 minutes timeout.
+                         */
+                        { client: client, timeout: 4 * 60 },
                         function onVmProvisioned(provisionErr) {
                             t.ifError(provisionErr,
                                 'VM should be provisioned successfully');
